force-app/main/default/classes/JoobleJSONCalloutMockGeneratorFailure.cls

```diff
@@ -0,0 +1,11 @@
+public with sharing class JoobleJSONCalloutMockGeneratorFailure implements HttpCalloutMock {
+    public HTTPResponse respond(HTTPRequest req) {
+        HttpResponse res = new HttpResponse();
+
+        res.setHeader('Content-Type', 'application/json');
+
+        res.setStatusCode(500);
```

Comment:
Great including a mock failed response too! You might be able to reduce code by having one mock generator that accepts a boolean for successResponse or something